### PR TITLE
SDESK-1816 As a admin I want to have a setting for "Publish Queue" expiry

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: apps/archive/commands.py [hunk 1]
```diff
@@ -48,6 +52,14 @@ def run(self):
 
         remove_locks()
 
+    def _remove_expired_publish_queue_items(self):
+        expire_interval = app.config.get('PUBLISH_QUEUE_EXPIRY_TIME', 0)
+        if expire_interval:
+            expire_time = utcnow() - timedelta(seconds=expire_interval)
```
Comment (petrjasek) [inline apps/archive/commands.py]: imo would be good to be consistent with the other options so the config should be in minutes

Comment (petrjasek) [reply apps/archive/commands.py]: like `PUBLISH_QUEUE_EXPIRY_MINUTES`

Comment (mugurrus) [reply apps/archive/commands.py]: Then I would have to set the sleep time for the behave test to 61 seconds, otherwise it wouldn't work.

Comment (petrjasek) [reply apps/archive/commands.py]: I think you can set an _id manually when you post to publish queue

Comment (mugurrus) [reply apps/archive/commands.py]: But this has nothing to do with _id

Comment (naspeh) [reply apps/archive/commands.py]: I agreed with @petrjasek regarding the name

Probably you can use `PUBLISH_QUEUE_EXPIRY_MINUTES = -1` for tests like https://github.com/superdesk/superdesk-core/blob/master/tests/audit_purge_test.py#L16

Comment (naspeh) [reply apps/archive/commands.py]: using this [superdesk.tests.steps.given_config_update](https://github.com/superdesk/superdesk-core/blob/master/superdesk/tests/steps.py#L376)

Comment (petrjasek) [reply apps/archive/commands.py]: if the _id is older it will pick it or am I missing something?

File: superdesk/default_settings.py [hunk 2]
```diff
@@ -618,3 +618,5 @@ def local_to_utc_hour(hour):
 # SAML Auth settings
 SAML_PATH = env('SAML_PATH')
 SAML_LABEL = env('SAML_LABEL', 'Single Sign On')
+
+PUBLISH_QUEUE_EXPIRY_TIME = env('PUBLISH_QUEUE_EXPIRY_TIME', 864000)
```
Comment (petrjasek) [inline superdesk/default_settings.py]: imo it should be 0 by default

**Pull request merged**